src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -337,18 +288,21 @@ void Tiltrotor::update_transition_state()
 		int rear_value = (1.0f - time_since_trans_start / _params_tiltrotor.front_trans_dur_p2) *
 				 (PWM_DEFAULT_MAX - PWM_DEFAULT_MIN) + PWM_DEFAULT_MIN;
 
-		set_rear_motor_state(VALUE, rear_value);
+
+		_motor_state = set_motor_state(_motor_state, VALUE, rear_value);
```

Comment:
This is kind of unfortunate, but I guess we should preserve the existing behaviour for now.